saleor/graphql/account/utils.py

```diff
@@ -204,6 +204,12 @@ def can_user_manage_group(user: "User", group: Group) -> bool:
     return user.has_perms(permissions)
 
 
+def can_manage_service_account(user: "User", service_account: "ServiceAccount") -> bool:
+    """User can't manage service account with permission that is out of scope."""
```

Comment:
Out of scope = ?

Reply:
Out of scope means that you don't have specific permission.